downstream/modules/aap-on-azure/proc-azure-nw-private-deploy-ssh-tunnel.adoc

```diff
@@ -15,40 +16,46 @@ This approach is easier to implement on Linux and macOS host machines, but can b
 . Update your local hosts file so that the {PlatformNameShort} URLs route traffic to your local machine rather than the private IP that DNS records are configured with.
 Add the following line to your hosts file:
 +
+[subs="+quotes"]
 ----
-127.0.0.1 controller.aapusb7en3awafew.az.ansiblecloud.com
+127.0.0.1 controller.<__your_AAPonAzure_instance__>.az.ansiblecloud.com
 ----
 +
 The following example shows the line in a hosts file:
 +
+[subs="+quotes"]
 ----
 ##
 # Host Database
 #
 # localhost is used to configure the loopback interface
 # when the system is booting. Do not change this entry.
-##
-127.0.0.1	localhost
-255.255.255.255	broadcasthost
+#
+127.0.0.1       localhost
+255.255.255.255 broadcasthost
 ::1             localhost
 
-127.0.0.1 controller.aapusb7en3awafew.az.ansiblecloud.com
-----
+127.0.0.1 controller.<__your_AAPonAzure_instance__>.az.ansiblecloud.com
 
+----
 . Run an `ssh` command to establish the SSH tunnel.
+This requires root access on your local machine.
```

Comment:
```suggestion
. As a user with root privileges, run the `ssh` command to establish an SSH tunnel.
```

Reply:
You could probably mix it up a little more but I think it makes sense to include the root privileges as part of the step.

"This requires root access on your local machine." - With this construction the pronoun "this" is not clearly referenced in my opinion. Do you mean "this step", "this command", or something else? It's a minor but unreferenced antecedents are a real thing.

I would also `s/an ssh command/the ssh command/`. There is only one ssh command so I think a definite article is more appropriate.

Likewise `s/the SSH tunnel/an SSH tunnel/` because it does not yet exist and using "the" here suggests you are referring to something that already exists.